src/backend/distributed/deparser/ruleutils_12.c

```diff
@@ -7487,6 +7488,126 @@ pg_get_triggerdef_command(Oid triggerId)
 	bool prettyOutput = false;
 	return pg_get_triggerdef_worker(triggerId, prettyOutput);
 }
+char *
```

Comment:
We need 2 empty lines above here.

Our formatter ignores these files as they come mostly from postgres. However, lets keep some simple rules applied here.